### ISSUE TITLE
AttributeError on remote.name from nbdimeserver.py

### ISSUE THREAD

**Issue by cdeil** [jupyter/nbdime#304]
I'm getting this error:
```
$ nbdiff-web check.ipynb 
[I nbdimeserver:322] Listening on 127.0.0.1, port 51599
[I webutil:29] URL: http://127.0.0.1:51599/difftool
[E web:1590] Uncaught exception GET /difftool (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:51599', method='GET', uri='/difftool', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Host': '127.0.0.1:51599', 'Connection': 'keep-alive', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36', 'Upgrade-Insecure-Requests': '1', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8', 'Accept-Encoding': 'gzip, deflate, br', 'Accept-Language': 'en-US,en;q=0.8,de;q=0.6,sv;q=0.4'})
Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/tornado/web.py", line 1509, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/Users/deil/Library/Python/3.6/lib/python/site-packages/nbdime/webapp/nbdimeserver.py", line 136, in get
    args["remote"] = remote.name
AttributeError: 'str' object has no attribute 'name'
[E web:2063] 500 GET /difftool (127.0.0.1) 5.62ms
[64446:38147:1017/171417.546759:ERROR:browser_gpu_channel_host_factory.cc(103)] Failed to launch GPU process.
[W web:2063] 404 GET /favicon.ico (127.0.0.1) 0.86ms
```
This is with nbdime 0.3 on Mac:
```
 pip install --user nbdime
Collecting nbdime
  Downloading nbdime-0.3.0-py2.py3-none-any.whl (1.7MB)
    100% |████████████████████████████████| 1.8MB 583kB/s 
Collecting colorama (from nbdime)
  Downloading colorama-0.3.9-py2.py3-none-any.whl
Requirement already satisfied: six in /opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from nbdime)
Requirement already satisfied: nbformat in /opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from nbdime)
Requirement already satisfied: tornado in /opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from nbdime)
Collecting GitPython (from nbdime)
  Downloading GitPython-2.1.7-py2.py3-none-any.whl (446kB)
    100% |████████████████████████████████| 450kB 2.3MB/s 
Requirement already satisfied: requests in /opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from nbdime)
Collecting gitdb2>=2.0.0 (from GitPython->nbdime)
  Downloading gitdb2-2.0.3-py2.py3-none-any.whl (63kB)
    100% |████████████████████████████████| 71kB 7.2MB/s 
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from requests->nbdime)
Requirement already satisfied: idna<2.7,>=2.5 in /opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from requests->nbdime)
Requirement already satisfied: urllib3<1.23,>=1.21.1 in /opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from requests->nbdime)
Requirement already satisfied: certifi>=2017.4.17 in /opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from requests->nbdime)
Collecting smmap2>=2.0.0 (from gitdb2>=2.0.0->GitPython->nbdime)
  Downloading smmap2-2.0.3-py2.py3-none-any.whl
Installing collected packages: colorama, smmap2, gitdb2, GitPython, nbdime
Successfully installed GitPython-2.1.7 colorama-0.3.9 gitdb2-2.0.3 nbdime-0.3.0 smmap2-2.0.3
```

**Comment by vidartf** [comment]
I can reproduce. Will look into it.

**Comment by vidartf** [comment]
Proposed fix in #307.
